shared/components/splash-example.jsx

```diff
@@ -39,18 +44,15 @@ export default React.createClass({
                 <h2 className="splash-example-heading">Example</h2>
 
                 <div className="splash-example-output">
-                    <CSSTransitionGroup
```

Comment:
Removed the transition because it flashed every time the example updated; i.e., every second.